lms/static/js/edxnotes/plugins/accessibility.js

```diff
@@ -148,6 +148,12 @@ define(['jquery', 'underscore', 'annotator'], function ($, _, Annotator) {
                 case KEY.ESCAPE:
                     this.annotator.viewer.hide();
                     break;
+                // Gives focus to the show/hide notes button
+                case KEY.n:
+                    if (event.ctrlKey && event.altKey)  {
+                        $('.action-toggle-notes').focus();
```

Comment:
We already use [aria-live="polite"](https://github.com/edx/edx-platform/blob/jmclaus/exit-focus-highlighted-text/lms/templates/edxnotes/toggle_notes.html#L11), so the users will be informed about what is happening.